README.md

```diff
@@ -171,12 +171,20 @@ policy violations by filing an issue. This is the quickest and easiest way to st
 
 Effort: very easy 
 
-Steps:   
-1) Install the [Allstar app](https://github.com/apps/allstar-app) (choose "All
-Repositories" under Repository Access, even if you plan to disable Allstar on some repositories later)  
-2) Fork the [sample repository
-](https://github.com/jeffmendoza/dot-allstar-quickstart) (simply click "Fork" button in the top right corner of the linked page) 
-3) Rename your fork to be `.allstar` instead of `dot-allstar-quickstart`
+Steps:
+
+1.  Install the Allstar app
+    1.  [Open the installation
+        page](https://github.com/apps/allstar-app) and click Configure
+    1.  If you have multiple organizations, select the one you want to
+        install Allstar on
+    1.  Select "All Repositories" under Repository Access, even if you
+        plan to disable Allstar on some repositories later
+1.  Fork the sample repository
+    1.  [Open the sample repository](https://github.com/jeffmendoza/dot-allstar-quickstart)
+        and click "Fork" button in the top right corner
```

Comment:
I changed the sample repo to a template a little while ago. I believe the button is now "Use this template". Then for the next step, the name field is empty and the user just types in `.allstar`. Then click "Create repository from template"

Reply:
Looks good, thanks. Updated.

Do users need to keep the repo set as public? That's an option below the repo name field, and we should specify if so.

Reply:
They don't, it is up to them